src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -112,24 +112,28 @@ public function load($uuid)
                 $locale
             );
 
-            $targetWebspace = $this->webspaceManager->findWebspaceByKey($contentStructure->getWebspaceKey());
-            $security = $targetWebspace->getSecurity();
-            $system = $security ? $security->getSystem() : null;
-
-            if ($targetWebspace->hasWebsiteSecurity()
-                && $this->securityChecker
-                && !$this->securityChecker->hasPermission(
-                    new SecurityCondition(
-                        PageAdmin::SECURITY_CONTEXT_PREFIX . $contentStructure->getWebspaceKey(),
-                        $locale,
-                        SecurityBehavior::class,
-                        $uuid,
-                        $system
-                    ),
-                    PermissionTypes::VIEW
-                )
-            ) {
-                return null;
+            $targetWebspaceKey = $contentStructure->getWebspaceKey();
+
+            if ($targetWebspaceKey) {
```

Comment:
Maybe we should here also check if `contentStructure` is from the correct structure type `page`, else the security context could be called with a none page uuid which just returns a webspace key.

Reply:
So it should maybe be something like instead of check for `getWebspaceKey`:

```
if ('page' !== $contentStructure->getType()) {
    return $this->structureResolver->resolve($contentStructure);
}
```